config/initializers/figaro.rb

```diff
@@ -45,7 +45,9 @@
   'session_encryption_key',
   'session_timeout_in_minutes',
   'state_id_proofing_vendor',
-  'twilio_accounts',
+  'twilio_numbers',
+  'twilio_sid',
+  'twilio_auth_token',
```

Comment:
Would you mind adding the messaging service SID here? I neglected to do that in my PR 😳